### ISSUE TITLE
"Special Events & Parties" menu item no longer works

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#682]
**Shows & Events > Special Events & Parties** used to redirect to http://burlesque-expo.com/scheduler/view_list/Special and according to all of the page editing it still does.

However, when you click on **Special Events & Parties**, nothing happens.  This problem started when @bethlakshmi added two new sub-menus.

*EDIT:* Similarly, the "All Shows" which is supposed to display scheduler/view_list/Show, doesn't do anything when you click on it.  It's like they've stopped being menu items and just become headers.

It looks like the solution is either [here][http://docs.django-cms.org/en/2.4.2/getting_started/navigation.html] or [here][http://docs.django-cms.org/en/develop/how_to/menus.html]

As a temporary fix, I'm duplicating the malfunctioning levels to include the same functionality in the lower level menus.  This isn't the way I want to do this.

**Comment by bethlakshmi** [comment]
I'm changing this from a bug to an enhancement, because it's not an oversight, it's a limitation of me not being able to intuit what's wanted when you ask for submenus to work.

My basic problem is that we have clickable menus.  So the user goes to a menu item and clicks it and the menu opens a sub menu.
If they click it and the page loads the new page ... they will not get a submenu, they will get the reloaded page.

Perhaps there's some obvious way to make a menu item both clickable, and loading the new page, but I couldn't think of how, so I didn't do it.  Also, I followed the pattern we had set up on the main menu headers (those also do not load the page).

This could be addressed next year, but I need you to point me to an example of how you want the menus to work to both load a page and open submenus.
